src/mscorlib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -686,6 +686,115 @@ protected EventSource(EventSourceSettings settings, params string[] traits)
             Initialize(eventSourceGuid, eventSourceName, traits);
         }
 
+#if FEATURE_PERFTRACING
+        // Generate all EventPipeEvent instances which belong to this EventProvider
```

Comment:
Change comment to 
    
// Generate the serialized blobs that describe events for all strongly typed events (that is events that define strongly
// typed event methods.   Dynamically defined events (that use Write<T>) hare defined on the fly and are 
// handled elsewhere.